lib/locales/de.js

```diff
@@ -12,62 +12,62 @@ module.exports = {
+        'lon:': 'Lon.:',                                                                                    // traces/scattergeo/calc.js:49
+        'q1:': 'q1',                                                                                        // traces/box/calc.js:130
+        'q3:': 'q3',                                                                                        // traces/box/calc.js:131
+        'source:': 'Quelle:',                                                                               // traces/sankey/plot.js:140
+        'target:': 'Ziel:',                                                                                 // traces/sankey/plot.js:141
+        'lower fence:': 'Untere Schranke:',                                                                 // traces/box/calc.js:134
+        'upper fence:': 'Obere Schranke:',                                                                  // traces/box/calc.js:135
+        'max:': 'Max.:',                                                                                    // traces/box/calc.js:132
+        'mean ± σ:': 'Mittelwert ± σ:',                                                                     // traces/box/calc.js:133
+        'mean:': 'Mittelwert:',                                                                             // traces/box/calc.js:133
+        'median:': 'Median:',                                                                               // traces/box/calc.js:128
+        'min:': 'Min.:',                                                                                    // traces/box/calc.js:129
+        'Turntable rotation': 'Drehscheibenorbit',                                                          // components/modebar/buttons.js:288
+        'Toggle Spike Lines': 'Bezugslinien an-/abschalten',                                                // components/modebar/buttons.js:548
+        'open:': 'Eröffnung:',                                                                              // traces/ohlc/transform.js:136
+        'high:': 'Höchstkurs:',                                                                             // traces/ohlc/transform.js:137
+        'low:': 'Tiefstkurs:',                                                                              // traces/ohlc/transform.js:138
+        'Toggle show closest data on hover': 'Anzeige der nähesten Daten an-/abschalten',                   // components/modebar/buttons.js:353
+        'incoming flow count:': 'Anzahl eingehender Verbindungen',                                          // traces/sankey/plot.js:142
+        'outgoing flow count:': 'Anzahl ausgehender Verbindungen',                                          // traces/sankey/plot.js:143
+        'kde:': 'Kerndichteschätzung'                                                                       // traces/violin/calc.js:73
```

Comment:
@jdugge:
'Dichte: ' (or more accurately 'Dichteschätzung: ', or even more accurately 'Kerndichteschätzung: '). Unfortunately, there doesn't seem to be a common abbreviation for this term in German.

Reply:
As this is used as a hover label prefix, I'd tend to place a premium on keeping it short. If you think a violin plot user (someone who already has a basic understanding of what information a violin plot conveys) who sees a point on the curve labeled with `'Dichte:'` would understand it to mean the kernel density estimate, then I'd vote for that instead.